lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -20,29 +20,21 @@ using namespace Windows::Globalization;
 
 #ifdef INTL_ICU
 #include <CommonPal.h>
-#include "PlatformAgnostic/IPlatformAgnosticResource.h"
-using namespace PlatformAgnostic::Resource;
 #include "PlatformAgnostic/ICU.h"
 using namespace PlatformAgnostic::ICUHelpers;
 
+template<typename ICUFunc>
+static void RecyclerExecuteICUWithRetry(_In_ ICUFunc func, _In_ Recycler *recycler, _Out_writes_opt_(returnLen) char16 **ret, _Out_ int *returnLen, _In_ int firstTryLength = 32)
+{
+    bool success = ExecuteICUWithRetry<char16, ICUFunc>([&](int length) -> char16 * {
```

Comment:
>-> char16 * [](start = 72, length = 11)

This should be unnecessary as it would be inferred by the compiler, right?

Reply:
I didn't try, I remember having issues back at school where gcc, at least, required return types if it wasn't void. I might be mis-remembering, though -- I can play around with it.